### ISSUE TITLE
Spot Check: Sims with aria-live content

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#681]
For phetsims/chipper#472, we are going to add some content to the top level html file for accessibility.  This content has been added in the following dev versions, where the aria-live text content has been populated with small, medium and very long strings:
 - http://www.colorado.edu/physics/phet/dev/html/rutherford-scattering/1.1.0-dev.1/rutherford-scattering_en.html
 - http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.6/john-travoltage_en.html?accessibility

Could you please check these versions quickly on a few platforms to make sure everything seems OK?  This change should not interfere with any sim specific features, just the following one each platform should be sufficient:
- Make sure the sim loads
- Play with the sim for a minute
- Cycle through all keyboard navigation (john-travoltage)

Assigning to @ariel-phet to prioritize.

**Comment by phet-steele** [comment]
These links work on Win 10 Edge, iOS 9.3.4, and OSX 10.11.6 Chrome. However, there is no aria-live content in an iframe!!
